### DISCUSSION TITLE
Feature "Discover"

### DISCUSSION THREAD

**Discussion by Dvalin21** [doitandbedone/ispyagentdvr-docker#448] [Q&A]
What does Ispy Agent use to pull the urls, camera options, and so forth when you click the button Discover? Thanks in advance.

**Comment by ispysoftware** [comment]
[onvif](https://github.com/ispysoftware/iSpy/tree/master/Onvif)

**Reply by Dvalin21** [reply]
Thank you very much. My camera is a ONVIF, but has rtsp url except the manufacturer closed port 554 and opened 2600 for rtsp. Port 80 is for the ONVIF. When I do a brute force on http with nmap I get "/" has no authentication. So you would think I would be able to get the ONVIF function working. Im assuming its what the app is using.
